spicy/src/compiler/codegen/unit-builder.cc

```diff
@@ -132,7 +132,7 @@ struct FieldBuilder : public hilti::visitor::PreOrder<void, FieldBuilder> {
 
         addHookImplementation(h.hook(), AttributeSet::find(h.hook().attributes(), "foreach"));
     }
-};
+}; // namespace
```

Comment:
Was this generated by clang-format (it does generate these comments). This paren terminates a struct, not a namespace.
```suggestion
};
```

Reply:
(Not sure.)